.travis.yml

```diff
@@ -69,6 +71,36 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+   - language: android
+     dist: trusty
+     android:
+       components:
+        - build-tools-26.0.1
+        # Minimum API level supported
+        - android-21
+     install:
+      # Install rust manually
+      - curl https://build.travis-ci.org/files/rustup-init.sh -sSf | sh -s -- -y --default-toolchain stable
+      - export PATH=$HOME/.cargo/bin:$PATH
+      - rustup default stable
+      - rustup target add aarch64-linux-android arm-linux-androideabi armv7-linux-androideabi i686-linux-android
+      # Install android ndk
+      - NDK_VER=r13b
```

Comment:
Same about `env` https://docs.travis-ci.com/user/environment-variables/

Reply:
Moved some constants for NDK and its url to env/global section.